osu.Game/Overlays/BeatmapSet/AuthorInfo.cs

```diff
@@ -95,8 +103,12 @@ public AuthorInfo()
         [BackgroundDependencyLoader(true)]
         private void load(UserProfileOverlay profile)
         {
-            this.profile = profile;
-            clickableArea.Action = () => profile?.ShowUser(avatar.User);
+            clickableArea.Action = () =>
+            {
+                if (avatar.User != null) profile?.ShowUser(avatar.User);
```

Comment:
avatar can be null

Reply:
it's set in the ctor

Reply:
Set BeatmapSet to null -> Click -> avatar == null

Reply:
avatar.User